src/tools/crawl.ts

```diff
@@ -30,6 +37,7 @@ export async function crawl(suite: string): Promise<IGroupDesc[]> {
       });
     } else if (path.basename(file) === 'README.txt') {
       const readme = file;
+      // Shouldn't this always be the case since glob found it?
```

Comment:
oh, yes. I used to glob for directories instead of READMEs.

Reply:
Done